src/System.Linq/tests/MinMaxTests.cs

```diff
@@ -0,0 +1,373 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public class MinMaxTests
+    {
+        [Fact]
+        public void MinInt()
+        {
+            var one = Enumerable.Range(1, 10).ToArray();
```

Comment:
Nit: how about just writing it out like you've done for the other arrays?  It would help with consistency for someone reading along.